src/bionics.cpp

```diff
@@ -1073,23 +1073,17 @@ bool player::has_enough_anesth( const itype *cbm )
         return true;
     }
 
-    const int difficulty = cbm->bionic->difficulty;
-    int amount = difficulty * 40;
+    const int weight = units::to_kilogram( g->u.bodyweight() ) / 10;
```

Comment:
Regarding `g->u`: isn't now possible to install bionics in NPCs? Should this not use their weight in that case?

Reply:
Yes, it's a mistake. Should be fixed now.